tracer/build/_build/Build.Utilities.cs

```diff
@@ -452,6 +453,35 @@ private static MSBuildTargetPlatform GetDefaultTargetPlatform()
         return MSBuildTargetPlatform.x64;
     }
 
+    [CanBeNull]
+    private static string GetDefaultRuntimeIdentifier()
```

Comment:
We generally don't use the `JetBrains.Annotations`... I'm not against it per-se, but we can use nullable refs instead?

Reply:
Weird. I think Rider converted `string?` to this because nullable reference types are not enabled. I just removed the attribute for now.